src/main/java/org/testng/internal/ClassBasedParallelWorker.java

```diff
@@ -9,14 +9,17 @@
 import org.testng.xml.XmlSuite;
 import org.testng.xml.XmlTest;
 
+import java.util.Arrays;
 import java.util.Collections;
 import java.util.List;
 import java.util.Map;
 import java.util.Set;
+import java.util.stream.Collectors;
 
 class ClassBasedParallelWorker extends AbstractParallelWorker {
 
     @Override
+    @SuppressWarnings("unchecked")
```

Comment:
It always smells. Where is the warning located?

Reply:
Fixed.